docs/custom_quantum_functions.md

```diff
@@ -90,68 +90,68 @@ keyrecord_t record {
 
 # LED Control
 
-QMK provides methods to read the 5 LEDs defined as part of the HID spec:
+QMK provides methods to read 5 of the LEDs defined in the HID spec:
 
-* `USB_LED_NUM_LOCK`
-* `USB_LED_CAPS_LOCK`
-* `USB_LED_SCROLL_LOCK`
-* `USB_LED_COMPOSE`
-* `USB_LED_KANA`
+* Num Lock
+* Caps Lock
+* Scroll Lock
+* Compose
+* Kana
 
-These five constants correspond to the positional bits of the host LED state.
 There are two ways to get the host LED state:
 
-* by implementing `led_set_user()`
-* by calling `host_keyboard_leds()`
+* by implementing `led_update_kb()` or `_user()`; or
+* by calling `host_keyboard_led_state()`
 
-## `led_set_user()`
+## `led_update_user()`
 
-This function will be called when the state of one of those 5 LEDs changes. It receives the LED state as a parameter.
-Use the `IS_LED_ON(usb_led, led_name)` and `IS_LED_OFF(usb_led, led_name)` macros to check the LED status.
+This function will be called when the state of one of those 5 LEDs changes. It receives the LED state as a struct parameter.
 
-!> `host_keyboard_leds()` may already reflect a new value before `led_set_user()` is called.
+You must return either `true` or `false` from this function, depending on whether you want to override the keyboard-level implementation.
 
-### Example `led_set_user()` Implementation
+!> `host_keyboard_led_state()` may already reflect a new value before `led_update_user()` is called.
+
+### Example `led_update_user()` Implementation
 
 ```c
-void led_set_user(uint8_t usb_led) {
-    if (IS_LED_ON(usb_led, USB_LED_NUM_LOCK)) {
+bool led_update_user(led_t led_state) {
```

Comment:
does this go in keyboard.c or keymap.c?

Reply:
It's under the "Function Documentation" heading - just following the format of this page.